internal/web/server.go

```diff
@@ -392,25 +417,37 @@ func (s *server) packageDiscussion(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	pkgName := mux.Vars(r)["pkgName"]
-	pkg, status, manifest, _, err := describe.DescribePackage(r.Context(), pkgName)
-	if err != nil {
-		err = fmt.Errorf("An error occurred fetching package details of %v: %w\n", pkgName, err)
+	repositoryName := mux.Vars(r)["repositoryName"]
+	pkg, manifest, err := describe.DescribeInstalledPackage(r.Context(), pkgName)
```

Comment:
I noticed two issues here during testing: when opening the discussion page for a not-installed package, the page stays empty because of a template error:

```
Unexpected error rendering package-detail (cyclops): template: base.html:71:30: executing "base.html" at <.Error>: Error is not a method but has arguments
 – This is most likely a BUG – Please report it here: https://github.com/glasskube/glasskube
```

* `err` is not `nil` because it's a not found error, and consequently the template gets the `Error` variable set (see `enrichWithErrorAndWarnings`). 
* The template parsing error seems to be a different issue that I don't understand yet, maybe it has something to do with the template changes, but I don't really know how that would be a problem.

Reply:
Ah I see, should probably add something like `if IsNotFound(err) { err = nil }`